sunkit_image/enhance.py

```diff
@@ -158,3 +165,95 @@ def mgn(
     image += Cprime_g
 
     return image
+
+
+def wow(*args, **kwargs):
+    """
+    Processes an image with the Wavelets Optimized Whitening (WOW) algorithm.
+
+    This function manipulates the wavelet spectrum of the input image so that the power in the output image
+    is equal at all locations and all spatial scales, thus 'whitening' the wavelet spectrum. By doing so, the
+    large scale structures are attenuated and the small scale structures are reinforced, with the objective criterion
+    that the variance at all scales must be equal. The algorithm has the added advantage to allow attenuation of the
+    noise at the same time, thus avoiding the usual explosion of noise usually inherent to enhancement methods.
+
+    .. note::
+        * The filter was developed with the intention to find an objective criterion for enhancement. By default, there
+          are therefore no free parameters to adjust, except for the denoising coefficients. It is however possible to
+          set the synthesis weights to values other than 1 to tune the output. It is also possible to merge the output
+          with a gamma-scaled version of the original image. The weight of the gamma scaled image can be adjusted using
+          the parameter 'h'.
+          to the type of input image (e.g. wavelength or channel). For most purposes, the weights can be set
+          equal for all scales.
+        * We don't deal with NaN (Not a Number) in this implementation.
+
+    Parameters
+    ----------
+    data : `numpy.ndarray`
+        Image to be transformed.
```

Comment:
```suggestion
    data : `sunpy.map.Map`
        Image to be transformed.
```
I hope that's the correct link.
Going forward, we want to make sure that functions in sunkit-image only work on sunpy.Maps. 

You can add this to handle this for you: https://github.com/sunpy/sunkit-image/blob/main/sunkit_image/utils/decorators.py#L9

Reply:
Done, thanks, I did not know that